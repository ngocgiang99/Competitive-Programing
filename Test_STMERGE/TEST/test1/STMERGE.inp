5
3 2
813220162 648761648 
140492042 606526283 
351769351 775455101 
7 9
421079661 644027123 695829443 403986505 404770857 359929047 397413730 131137168 965691696 
751281836 212936670 969635876 55773367 895451704 50412172 250058474 33830450 260765569 
463057154 318920455 351720280 593295562 445861652 860079327 677053980 764015770 228988632 
799437592 201104052 870762933 555035298 937466518 256328597 333126944 354480903 741825663 
541392244 563213345 856416477 261243448 955895110 639641659 478193464 394888037 41605696 
806447413 372687726 964376044 470354903 698594066 376430355 274391026 49144767 621217817 
932023292 948515803 737109487 773736654 830654015 997007949 632643226 86823414 834814512 
10 2
778093718 238304869 
544066207 784192377 
868642292 830813106 
794140424 858181462 
266397075 976271148 
290309903 449220866 
487319866 963319849 
537566871 595164660 
971074797 872659517 
489701318 499860448 
1 4
251052800 53151419 972694475 679792447 
3 3
648077376 777932952 691139692 
805349968 965197653 116136923 
931737767 553684534 223537744 
