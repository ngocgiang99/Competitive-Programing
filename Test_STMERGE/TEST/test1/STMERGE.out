351769351
484237144
238304869
251052800
691139692
